experiment/measurer/run_crashes.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for processing crashes."""
+
+import collections
+import os
+
+from clusterfuzz import stacktraces
+
+from common import logs
+from common import new_process
+from experiment.measurer import sanitizer
+
+logger = logs.Logger('run_crashes')
+
+# Memory limit for libfuzzer crash.
+RSS_LIMIT_MB = 2048
```

Comment:
Isn't this a duplicate of the same thing from run_coverage? please use one source of truth.

Reply:
Done!